.github/workflows/build-assets-on-release.yml

```diff
@@ -0,0 +1,72 @@
+name: Build assets on release and create archive
+on: 
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+          node: [14]
+
+    steps:
+    - name: 'Checkout master branch'
+      uses: actions/checkout@master
+
+    - name: PHP setup
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.4
+        coverage: pcov
```

Comment:
Why do you need coverage for a build process?

Reply:
C/P from our other projects, forgot to remove 🙈  removed!